src/World.cpp

```diff
@@ -1042,9 +1044,60 @@ void cWorld::Tick(std::chrono::milliseconds a_Dt, std::chrono::milliseconds a_La
 		SaveAllChunks();
 	}
 
-	if (m_WorldAge - m_LastUnload > std::chrono::seconds(10))  // Unload every 10 seconds
+
+	// See if we should unload chunks. Cuberite likes to keep as many chunks as possible in RAM for maximum performance.
+	// Chunks are unloaded only when the allocated chunk space (configurable in settings.ini) is full.
+	if (m_WorldAge - m_LastUnload > std::chrono::seconds(2))
 	{
-		UnloadUnusedChunks();
+		cRoot * Root = cRoot::Get();
+		int TotalChunks = Root->GetTotalChunkCount();           // The amount of loaded chunks in all worlds combined
+		int MyChunks = GetNumChunks();                          // Our amount of loaded chunks
+		int MaxChunks = Root->GetMaxAllowedChunks();            // The maximum amount of allowed loaded chunks
```

Comment:
I don't think admins should care about optimizing to such a resolution; specify the max amount of loaded chunks, and let Cuberite naturally handle the balance in accordance with how many chunks/players are at each world. In your example, the lobby, being smaller and since players are all in one area,  will have few chunks loaded, while the main world will have a bigger piece of the cache. This is pretty good.